tests/foreman/cli/test_hostgroup.py

```diff
@@ -316,21 +316,28 @@ def test_positive_create_with_multiple_entities(self):
             'organization-ids': org['id'],
         })
 
-        hostgroup = make_hostgroup({
+        make_hostgroup_params = {
             'location-ids': loc['id'],
             'environment-id': env['id'],
-            'lifecycle-environment': lce['name'],
             'puppet-proxy-id': puppet_proxy['id'],
             'puppet-ca-proxy-id': puppet_proxy['id'],
             'content-view-id': cv['id'],
             'domain-id': domain['id'],
             'subnet-id': subnet['id'],
-            'organization-id': org['id'],
+            'organization-ids': org['id'],
             'architecture-id': arch['id'],
             'partition-table-id': ptable['id'],
             'medium-id': media['id'],
             'operatingsystem-id': os['id'],
-        })
+        }
+        # If bug is open provide LCE id as parameter
+        # because LCE name cause errors
+        if bz_bug_is_open(1395254):
+            make_hostgroup_params['lifecycle-environment-id'] = lce['id']
+        else:
+            make_hostgroup_params['lifecycle-environment'] = lce['name']
```

Comment:
nitpick: i'd rather return back `lce['name']` to the list of parameters, and if bug is opened i'd pop it & used `id` instead. The reason is simple - you'll design the test in the way it supposed to work and the workaround will be applied only in case bug is open, right now workaround for params is applied regardless of bug state :) And also it adds readability as it'll become clear which params are supposed to be passed to `make_hostgroup()`